stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -889,6 +889,10 @@ function(_compile_swift_files
         COMMENT "Generating ${module_file}")
 
     if(SWIFTFILE_STATIC)
+      set(command_copy_interface_file)
+      if("${interface_file}")
+        set(command_copy_interface_file COMMAND "${CMAKE_COMMAND}" "-E" "copy" ${interface_file} ${interface_file_static})
```

Comment:
I don't understand why this is suddenly conditionalized.  What am I not understanding?

Reply:
With Library Evolution off we don't produce the .swiftinterface file at all, `interface_file` ends up empty and the "-E copy" command is ill-formed.